pyproject.toml

```diff
@@ -45,8 +45,8 @@ tqdm = ">=4,<5"
 Pillow = "^10.0.1"
 retrying = "^1.3.3"
 Shapely = [
-    { version = "^1.8.5", python = ">=3.8,<3.11" },
-    { version = "^2", python = ">=3.11" }
+    { version = "^1.8.5", python = ">=3.8,<3.12" },
+    { version = "^2", python = ">=3.12" }
```

Comment:
Does `>=1.8.5,<3` work as a pin here? Prefer to avoid splitting version dependencies unless necessary